pkg/feeds/pypi/pypi_artifacts_test.go

```diff
@@ -403,10 +403,14 @@ func TestPyPIArtifactsLive(t *testing.T) {
 	}
 
 	cutoff := time.Now().AddDate(0, 0, -1)
-	pkgs, errs := feed.Latest(cutoff)
+	pkgs, gotCutoff, errs := feed.Latest(cutoff)
 	if len(errs) != 0 {
 		t.Fatalf("feed.Latest returned error: %v", err)
 	}
+	wantCutoff := time.Now().UTC()
+	if gotCutoff.Sub(wantCutoff).Abs() > 5*time.Minute {
```

Comment:
why is it 5 minutes here when the other times are 1 second?

Reply:
I fixed this. The test was pulling live data, so I now just check that it doesn't match the input `cutoff`.